src/state.ts

```diff
@@ -0,0 +1,59 @@
+import {Channel, getChannelId} from './channel';
+import {Outcome, encodeOutcome, hashOutcome} from './outcome';
+import {Uint256, Address, Bytes, Bytes32} from './types';
+import abi from 'web3-eth-abi';
+import {keccak256, defaultAbiCoder} from 'ethers/utils';
+
+export interface State {
+  turnNum: number;
+  isFinal: boolean;
+  channel: Channel;
+  challengeDuration: string;
```

Comment:
This is something I've been wondering about. I expect `State` to be used by consumers of the library and I'm wondering if a type alias of `uint256` is more confusing than a `string`?

`getFixedPart` will only be internally used so that's why I went with `uint256` there. However it would be good to be consistent across the board.